Upstatement/ups-public-library
# Update temples

Hi ho! I released [temples v2](https://www.npmjs.com/package/temples) some months ago with much more stable API and new features (e.g. inserting into files, customizing Handlebars instance). Nothing major that would apply to pub lib but we should update.

## Changes
- Update temples to v2.1.1
- Simplify `.temples.yaml` file
- Change template extensions from `.template` to `.hbs`

## Steps To Test
1. Run `npm run temples`
1. Make sure each generator still pulls from the same templates folder and outputs the correct files.
